include/seqan3/range/views/interleave.hpp

```diff
@@ -54,17 +54,20 @@ template <std::ranges::random_access_range urng_t, std::ranges::random_access_ra
 class view_interleave : public std::ranges::view_interface<view_interleave<urng_t, inserted_rng_t>>
 {
 private:
+
     //!\brief The underlying range.
```

Comment:
```suggestion
private:
    //!\brief The underlying range.
```
no newlines **after** `public/private/protected` keyword (but before is correct). Tiny :nail_care: